projects/packages/status/src/class-modules.php

```diff
@@ -238,8 +242,20 @@ public function get_slug( $file ) {
 	public function get_available( $min_version = false, $max_version = false, $requires_connection = null, $requires_user_connection = null ) {
 		static $modules = null;
 
-		if ( ! Constants::is_defined( 'JETPACK__VERSION' ) || ! Constants::is_defined( 'JETPACK__PLUGIN_DIR' ) ) {
-			return array();
+		if ( ! class_exists( 'Jetpack' ) || ! Constants::is_defined( 'JETPACK__VERSION' ) || ! Constants::is_defined( 'JETPACK__PLUGIN_DIR' ) ) {
+			return array_unique(
+				/**
+				 * Stand alone plugins need to use this filter to register the modules they interact with.
+				 * This will allow them to activate and deactivate these modules even when Jetpack is not present.
+				 *
+				 * @since $$next-version$$
+				 *
+				 * @param array $modules The list of available modules as an array of slugs.
+				 * @param bool $requires_connection Whether to list only modules that require a connection to work.
+				 * @param bool $requires_user_connection Whether to list only modules that require a user connection to work.
+				 */
+				apply_filters( 'jetpack_get_available_standalone_modules', array(), $requires_connection, $requires_user_connection )
```

Comment:
Just a note: The reason I was using the same existing `jetpack_get_available_modules` filter before was to respect the environments that were already filtering it out for some reason. I don't know how likely it is that VIP/atomic/whoever would want to continue filtering these if their users explicitly have the standalone plugin... probably not very.